test/components/shareville-logo/logo.test.js

```diff
@@ -11,7 +11,7 @@ describe('<SharevilleLogo />', () => {
   });
 
   it('should render a SharevilleLogo', () => {
-    expect(wrapper.find('.sharevilleLogoWTClass-1')).to.have.length(1);
+    expect(wrapper.find('.svLogoGradientWtClass-1')).to.have.length(1);
```

Comment:
nay

Reply:
please use mockClasses https://github.com/nordnet/nordnet-ui-kit/blob/master/docs/testing-component.md#shallow

Reply:
@iamstarkov can't, these are hard-coded classes from the generated svg